src/components/SpecMetaData/utils.ts

```diff
@@ -34,7 +34,12 @@ export const getTestsSteps = (data: IEnvironmentData, fileID: string, testID: st
         status: test.status,
         ...(test.result?.message && { message: test.result.message }),
         ...(test.result?.beautifiedStack && { stacktrace: test.result.beautifiedStack }),
-        ...(test.result?.message && { shortMessage: test.result.message.split('-') }),
+        ...(test.result?.message && {
+          shortMessage: [
+            test.result.message.split(' - ')[0],
+            test.result.message.split(' - ').slice(1).join(' - '),
```

Comment:
is this change needed?

Reply:
This is the main change in this PR to fix the linked issue. It should only split on the first occurrence of ` - ` and not after that, otherwise the error message gets truncated.

Reply:
I've checked this change for possible errors and it will never throw an error for any string values of `this.result.message`.